### PR TITLE
databases-week3/NicoQ

### PR DESCRIPTION
**Pull request by nicoquiroga90**
Thanks for the review

### CODE REVIEW

File: 5_databases/week3/Homework.sql [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+SELECT * 
+FROM `Review`
+WHERE id=1;
+
+UPDATE `Review`
+SET stars =5
+WHERE id=1;
+
+DELETE from `Review`
+WHERE id=1;
```
Comment (BJORN-VINTHER) [inline 5_databases/week3/Homework.sql:91]: Almost all the queries are perfect 🦸‍♂️

File: 5_databases/week3/Homework.sql [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+SELECT *
+FROM Meal
+WHERE title LIKE "%persille%" ;
+
+
+-- Get meals that has been created between two dates
+SELECT *
+FROM Meal
+WHERE  Meal.created_date >= '2020-01-01'
+AND Meal.created_date <= '2023-11-05';
```
Comment (BJORN-VINTHER) [inline 5_databases/week3/Homework.sql:148]: Your query solves the problem. MySQL has a neat syntax for searching in a date range:
``` sql
SELECT *
FROM meal
WHERE created_date BETWEEN '2023-10-01' AND '2023-10-31';
```

File: 5_databases/week3/Homework.sql [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+JOIN Review ON Meal.id = Review.meal_id
+WHERE Review.stars > 3
+
+
+-- Get reservations for a specific meal sorted by created_date
+SELECT *
+FROM Meal
+JOIN Reservation ON Meal.id = Reservation.meal_id
+WHERE Meal.title LIKE "Mila%"
+ORDER BY Reservation.created_date ASC;
```
Comment (BJORN-VINTHER) [inline 5_databases/week3/Homework.sql:169]: You are asked to get the reservations for a meal, but here you return the meal. How can you modify the query to return the reservations?

File: 5_databases/week3/Homework.sql [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+
+
+-- Sort all meals by average number of stars in the reviews
+SELECT Meal.title, AVG(Review.stars)
+FROM Meal
+JOIN Review ON Meal.id = Review.meal_id
+GROUP BY Meal.title
+ORDER BY AVG(Review.stars) DESC;
```
Comment (BJORN-VINTHER) [inline 5_databases/week3/Homework.sql:177]: A user calls you and says that she doesn't get all the meals when using this query. It seems that most of the newly added meals don't appear. What could be the issue? 😮

**Review by BJORN-VINTHER — COMMENTED** [review]
Good job Nico, It looks like you're ready to become a database master 💯

File: 5_databases/week3/Homework.sql [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+
+     CREATE TABLE
+    `Review` (
+        `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+        `title` VARCHAR(255) NOT NULL,
+        `description` TEXT,
+        `meal_id` INT(10),
+        `stars` INT(10), 
+        `created_date` DATE
+    ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
```
[the remark this answers is not in the document]
Comment (nicoquiroga90) [reply 5_databases/week3/Homework.sql]: Cool, it makes sense, thanks!

File: 5_databases/week3/Homework.sql (hunk 2 above)
Comment (nicoquiroga90) [reply 5_databases/week3/Homework.sql]: Super

File: 5_databases/week3/Homework.sql (hunk 4 above)
Comment (nicoquiroga90) [reply 5_databases/week3/Homework.sql]: Oh, okey, im missing data of the meals without associated reviews, I will fix it. Thanks for the entire review 👍

**Pull request closed without merging**